src/app/container/version-modal/version-modal.component.html

```diff
@@ -355,31 +355,31 @@ <h4 mat-dialog-title>{{ TagEditorMode[mode] }} Version Tag</h4>
               </div>
             </div>
           </div>
-          <div class="form-group form-group-sm" *ngIf="!unsavedVersion.create">
+          <div class="form-group form-group-sm">
             <label class="col-sm-3 col-md-3 col-lg-3 control-label"> Image Size: </label>
             <div class="col-sm-9 col-md-9 col-lg-9">
               <div class="label-value">
                 {{ unsavedVersion.size ? getSizeString(unsavedVersion.size) : 'n/a' }}
               </div>
             </div>
           </div>
-          <div class="form-group form-group-sm" *ngIf="!unsavedVersion.create">
+          <div class="form-group form-group-sm">
             <label class="col-sm-3 col-md-3 col-lg-3 control-label"> Last Modified: </label>
             <div class="col-sm-9 col-md-9 col-lg-9">
               <div class="label-value">
-                {{ getDateTimeMessage(unsavedVersion.last_modified) }}
+                {{ getDateTimeMessage(unsavedVersion.last_built) }}
```

Comment:
Tags don't have last_modified, they have last_built. Thank you compiler. But, still, a little concerned.